### ISSUE TITLE
Support Bot Repo Stack Tracing

### ISSUE THREAD

**Issue by Skillz4Killz** [Skillz4Killz/Discordeno#56]
**Is your feature request related to a problem? Please describe.**
It would be nice to have error stacks from the bots own repo as opposed to the library. This helps determine what line of code from the user's own code is causing the error.

**Describe the solution you'd like**
The issue i made in deno has been solved and the PR is merged. It is releasing in 1.0.3 in deno and this needs to be added in.

**Comment by Skillz4Killz** [comment]
I think this should be implemented in v9. We still have time waiting for Discord to finalze their api and this is just sooo darn crucial to have imo

**Comment by Skillz4Killz** [comment]
Closed, will be in v9 #100 

**Previous Errors:**
```ts
error: Uncaught Error: REQUEST_CLIENT_ERROR
      throw new Error(Errors.REQUEST_CLIENT_ERROR);
            ^
    at handleStatusCode (requestManager.ts:314:13)
    at Object.callback (requestManager.ts:233:9)
    at async requestManager.ts:101:28
```

**New Errors:**
```ts
{
  type: "error",
  data: {
    method: "patch",
    url: "https://discord.com/api/v8/guilds/223909216866402304/members/130136895395987456",
    body: { nick: "test" },
    retryCount: 0,
    bucketID: undefined,
    errorStack: Error: Location In Your Files:
    at runMethod (requestManager.ts:201:9)
    at Object.patch (requestManager.ts:133:12)
    at editMember (member.ts:185:25)
    at Object.messageCreate (debug.ts:36:9)
    at Object.handleInternalMessageCreate [as MESSAGE_CREATE] (messages.ts:50:32)
  }
}
```
